packages/pathfinder/src/pathfinder.ts

```diff
@@ -248,8 +329,29 @@ export class Pathfinder {
 
         // todo implement the pathfinding logic
         // be sure to use REFERENCE_ACCOUNT_PUBLIC_KEY as the destination and source accounts
-
-        return Promise.resolve([]);
+        return server.paths(Pathfinder.REFERENCE_ACCOUNT_PUBLIC_KEY, 
```

Comment:
Too many uses of `any`. `any` should only be used as a last resort in case Typescript cannot feasibly resolve the type. None of the uses here match that criteria.

Reply:
resolved